public/apps/configuration/panels/permission-list/test/__snapshots__/permission-list.test.tsx.snap

```diff
@@ -7,6 +7,49 @@ exports[`Permission list page  AccessError component should load access error co
     coreStart={
       Object {
         "http": 1,
+        "uiSettings": Object {
+          "get": [MockFunction] {
```

Comment:
Why is the test creating this array?

Reply:
This is a code generated file based on the mock that it is passing.